apps/explorer/config/config.exs

```diff
@@ -88,6 +88,12 @@ config :explorer, Explorer.Staking.PoolsReader,
   validators_contract_address: System.get_env("POS_VALIDATORS_CONTRACT"),
   staking_contract_address: System.get_env("POS_STAKING_CONTRACT")
 
+config :explorer, Explorer.Celo.AccountReader,
+  accounts_contract_address: System.get_env("ACCOUNTS_CONTRACT_ADDRESS"),
```

Comment:
It would be easier to just set these variables the same time as we load the ABIs from gcloud if we are going to do that

Reply:
We are moving away from relying on the ABIs stored on gcloud and instead generating ABIs from a particular commit in the monorepo, like we do in ContractKit. Thus, contract addresses are runtime configuration that we are trying to centralize in the Registry if I'm not mistaken @mcortesi

Reply:
Hey, so it's what @nambrot says:

1) We should not download artifacts from glcoud
2) There's only one hardcoded address, the registry one, which remains constant across all environments.
3) For the other address we check the registry. `Registry.getAddressFor()`

There's also something to consider. **Contract & address inside registry can change due to governance proposals. So, we can't treat addresses or ABIs are constant through the life on an environment. We should be able to update those, or recheck them.**

Right now, contractkit is built to work with the contract ABI that live on the same commit on the protocol package. For that, it runs `yarn build` on protocol package which extracts the ABI. I'm not familiar with the current use case on this repo but happy to help devise a solution.

For the address, when creating a new contractkit instance, it will (lazily) fetch contract address on demand. But if we build a long running process that relies on contractkit, this process should be aware of the fact that address/contract might change... which is something we haven't done yet